### PR TITLE
Release bigquery 1.20.0

### PR DESCRIPTION
**Pull request by busunkim96**
This pull request was generated using releasetool.

### CODE REVIEW

**Comment by busunkim96** [comment]
@googleapis/api-bigquery Please close if this isn't a good time for a release. 

I am cutting this because the Yoshi team is releasing clients with the new endpoints (bigquery.googleapis.com instead of www.googleapis.com).

File: bigquery/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,32 @@
...
 [1]: https://pypi.org/project/google-cloud-bigquery/#history
 
+## 1.20.0
+
+09-13-2019 11:22 PDT
+
+
+### Implementation Changes
+- Change default endpoint to bigquery.googleapis.com ([#9213](https://github.com/googleapis/google-cloud-python/pull/9213))
+- Change the default value of Cursor instances' arraysize attribute to None ([#9199](https://github.com/googleapis/google-cloud-python/pull/9199))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Change the default value of Cursor instances' `arraysize` attribute to None ([#9199](https://github.com/googleapis/google-cloud-python/pull/9199))
```

File: bigquery/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,32 @@
...
+- Deprecate automatic schema conversion ([#9176](https://github.com/googleapis/google-cloud-python/pull/9176))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Deprecate automatic schema conversion. ([#9176](https://github.com/googleapis/google-cloud-python/pull/9176))
```

File: bigquery/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,32 @@
...
+- Fix list_rows() max results with BQ storage client ([#9178](https://github.com/googleapis/google-cloud-python/pull/9178))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Fix `list_rows()` max results with BQ storage client ([#9178](https://github.com/googleapis/google-cloud-python/pull/9178))
```

File: bigquery/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,32 @@
...
+
+### New Features
+- Add 'Model.encryption_config' (via synth) ([#9214](https://github.com/googleapis/google-cloud-python/pull/9214))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Add `Model.encryption_config`. (via synth) ([#9214](https://github.com/googleapis/google-cloud-python/pull/9214))
```

File: bigquery/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,32 @@
...
+- Add client.insert_rows_from_dataframe() method ([#9162](https://github.com/googleapis/google-cloud-python/pull/9162))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Add `Client.insert_rows_from_dataframe()` method ([#9162](https://github.com/googleapis/google-cloud-python/pull/9162))
```

File: bigquery/CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,32 @@
...
+- Add support for array parameters to Cursor.execute() ([#9189](https://github.com/googleapis/google-cloud-python/pull/9189))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Add support for array parameters to `Cursor.execute()`. ([#9189](https://github.com/googleapis/google-cloud-python/pull/9189))
```

File: bigquery/CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,32 @@
...
+- Add support of the project ID with org prefix to the Table.from_string() method ([#9161](https://github.com/googleapis/google-cloud-python/pull/9161))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Add support for project IDs with org prefix to `Table.from_string()` factory. ([#9161](https://github.com/googleapis/google-cloud-python/pull/9161))
```

File: bigquery/CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,32 @@
...
+- Add `--max_results` option to Jupyter magics ([#9169](https://github.com/googleapis/google-cloud-python/pull/9169))
+- Autofetch table schema on load if not provided ([#9108](https://github.com/googleapis/google-cloud-python/pull/9108))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Autofetch table schema on load if not provided. ([#9108](https://github.com/googleapis/google-cloud-python/pull/9108))
```

File: bigquery/CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,32 @@
...
+- Add `max_results` parameter to `QueryJob.result()` ([#9167](https://github.com/googleapis/google-cloud-python/pull/9167))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Add `max_results` parameter to `QueryJob.result()`. ([#9167](https://github.com/googleapis/google-cloud-python/pull/9167))
```

File: bigquery/CHANGELOG.md [hunk 10]
```diff
@@ -4,6 +4,32 @@
...
+
+### Documentation
+- Fix doc link ([#9200](https://github.com/googleapis/google-cloud-python/pull/9200))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Fix doc link. ([#9200](https://github.com/googleapis/google-cloud-python/pull/9200))
```

File: bigquery/CHANGELOG.md [hunk 11]
```diff
@@ -4,6 +4,32 @@
...
+
+### Internal / Testing Changes
+- Revert "Disable failing snippets test ([#9156](https://github.com/googleapis/google-cloud-python/pull/9156))" ([#9220](https://github.com/googleapis/google-cloud-python/pull/9220))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Revert "Disable failing snippets test ([#9156](https://github.com/googleapis/google-cloud-python/pull/9156))." ([#9220](https://github.com/googleapis/google-cloud-python/pull/9220))
```

**Comment by busunkim96** [comment]
@plamut Is it OK to release BigQuery?

**Comment by plamut** [comment]
@busunkim96 I don't know the roadmap, but it's probably fine. However, if not urgent, it would IMO make sense to wait a bit more for the currently open BigQuery PRs to get reviewed and merged (at least for those that don't require significant extra work).

**Comment by busunkim96** [comment]
@plamut Thank you! Justin's asked that this endpoint change land in the first half of the week, so I will proceed with the release.

**Pull request merged**